src/antlr/Parser.g

```diff
@@ -414,12 +414,12 @@ simpleUnaliasedSelector returns [Selectable.Raw s]
     ;
 
 selectionFunction returns [Selectable.Raw s]
-    : K_COUNT '(' '\*' ')'                      { $s = Selectable.WithFunction.Raw.newCountRowsFunction(); }
-    | K_MAXWRITETIME '(' c=sident ')'           { $s = new Selectable.WritetimeOrTTL.Raw(c, Selectable.WritetimeOrTTL.Kind.MAX_WRITE_TIME); }
-    | K_WRITETIME '(' c=sident ')'              { $s = new Selectable.WritetimeOrTTL.Raw(c, Selectable.WritetimeOrTTL.Kind.WRITE_TIME); }
-    | K_TTL       '(' c=sident ')'              { $s = new Selectable.WritetimeOrTTL.Raw(c, Selectable.WritetimeOrTTL.Kind.TTL); }
-    | K_CAST      '(' sn=unaliasedSelector K_AS t=native_type ')' {$s = new Selectable.WithCast.Raw(sn, t);}
-    | f=functionName args=selectionFunctionArgs { $s = new Selectable.WithFunction.Raw(f, args); }
+    : K_COUNT        '(' '\*' ')'                                    { $s = Selectable.WithFunction.Raw.newCountRowsFunction(); }
+    | K_MAXWRITETIME '(' c=sident m=selectorModifier[c] ')'          { $s = new Selectable.WritetimeOrTTL.Raw(c, m, Selectable.WritetimeOrTTL.Kind.MAX_WRITE_TIME); }
```

Comment:
now `maxwritetime` can be applied to a slice?

Reply:
Yes, `maxwritetime` can be applied to entire collections, slices and collection items :)

It's verified for all the test cases in `WritetimeOrTTLTest`, [here](https://github.com/apache/cassandra/blob/a1c6828d186ee7ac36e8965b70930341d1ea5196/test/unit/org/apache/cassandra/cql3/validation/entities/WritetimeOrTTLTest.java#L1086-L1091) and [here](https://github.com/apache/cassandra/blob/a1c6828d186ee7ac36e8965b70930341d1ea5196/test/unit/org/apache/cassandra/cql3/validation/entities/WritetimeOrTTLTest.java#L1116-L1122).